bundles/remote_services/discovery_shm/CMakeLists.txt

```diff
@@ -37,7 +37,9 @@ target_include_directories(rsa_discovery_shm PRIVATE
 		$<TARGET_PROPERTY:Celix::rsa_discovery_common,INCLUDE_DIRECTORIES>
 		$<TARGET_PROPERTY:Celix::civetweb,INCLUDE_DIRECTORIES>
 )
-target_link_libraries(rsa_discovery_shm PRIVATE Celix::framework CURL::libcurl ${LIBXML2_LIBRARIES})
+target_link_libraries(rsa_discovery_shm PRIVATE
+		Celix::framework CURL::libcurl ${LIBXML2_LIBRARIES} Celix::log_helper
```

Comment:
I suggest adding find module for libxml2.

Reply:
For libxml2, there is no problem using `LIBXML2_LIBRARIES`, since it is the same variable used in conan generated `conan-official-libxml2-variables.cmake`.